### PR TITLE
Finish Introduction tour

### PR DESCRIPTION
**Pull request by ccordoba12**
This PR adds the missing contents of the *Introduction tour*.

I decided to create it given that this was @goanpeca's work, so I'd like to know his thoughts about my additions.

### CODE REVIEW

**Comment by goanpeca** [comment]
Hey @ccordoba12 thanks for working on this. It looks fine on a quick look, but there is an issue that needs to be solved, maybe the ipython console is not intalled, so we need to include this in the logic of the tours.py

File: spyderlib/tour.py [hunk 1]
```diff
@@ -108,41 +118,72 @@ def get_tour(index):
...
+                           "<a href=\"%s\">here</a>.<br><br>"
+                           "It's also important to note that you can open as "
+                           "many consoles as you want, by going to the "
+                           "menu entry<br><br>"
+                           "<i>Consoles > Open an IPython console</i>.<br><br>"
+                           "Please click on the button below to run some simple "
+                           "code in this console. This will be useful to show "
+                           "you other important Spyder features." % \
+                           qtconsole_link),
+              'widgets': [sw.ipython_console],
```
Comment (goanpeca) [inline spyderlib/tour.py]: This will blow if ipython is not installed...

Comment (ccordoba12) [inline spyderlib/tour.py]: Yes, we need to verify that `ipython_console` exists before adding this part to the tour.

File: spyderlib/tour.py [hunk 2]
```diff
@@ -108,41 +118,72 @@ def get_tour(index):
...
+             {'title': _("The Python console"),
+              'content': _("You can also run your code on a <b>Python console</b>. "
+                           "These consoles are useful because they are created "
+                           "much faster than an IPython console, and let you "
+                           "run each file in a dedicated console<br><br>"
+                           "By pressing the button below, you will notice that "
+                           "<b>Python consoles</b> are also connected to the "
+                           "Variable Explorer, and that the last one only shows "
+                           "the variables of the currently focused console."),
+              'widgets': [sw.external_console, sw.variable_explorer],
```
Comment (goanpeca) [inline spyderlib/tour.py]: I would advice against this based on a suggestion by @Nodd.

Even though the functionality allows for highlighting any number of widgets at the same time, there might be the case where someone has the widgets in the same dockwidget location, in this case this lines will "fail" so it is better to use ONE and only ONE widget per slide. 

If you want to display how the variable explorer stored something... then it should be an additional slide

File: spyderlib/tour.py (hunk 1 above)
Comment (goanpeca) [inline spyderlib/tour.py]: @ccordoba12 can you please use `'Text {0}'.format(variable)` instead of `%`, I think is better and more flexible.

**Comment by goanpeca** [comment]
What should be the default behavior when a widget is not there? 
skip it? make it non-selectable? make it like grayed out in the combobox?

Depending on this I can make a PR to correct this, and also include the suggestion of @Nodd of using a dummy dockwidget instead of making it transparent to events (the temporal fix used right now...)

**Comment by Nodd** [comment]
It would be bad to skip it completely since the tour is there to show all features of spyder to new users.

Maybe add a warning that the corresponding widget it not active ?

**Comment by ccordoba12** [comment]
The only plugin that can be missing right now is the IPython console (which is loaded on demand). All other plugins (even the Pylint one) load fine.

But @Nodd has a good point: if the IPython Console is missing, the tour should show a warning and guide the user on how to install it.

**Comment by Nodd** [comment]
There is also the case where a panel is not enabled. Is this manager ? (if I remember well, @goanpeca told me it was working, but it was a long time ago)

**Comment by goanpeca** [comment]
If the panel is not enabled, it should be a problem... as far as I remember (it will be enabled and brought to the front)

Sure, we can then display a replacement content indicating how to install ipython.

File: spyderlib/tour.py (hunk 2 above)
Comment (ccordoba12) [inline spyderlib/tour.py]: Ok, I'll change it. But my idea would be to use the new custom layouts functionality to always show this tour uisng the *Spyder* layout.

But that's for another PR :-)

**Comment by ccordoba12** [comment]
intro I want to make clear that the purpose of this PR is only to add missing **contents** to the tutorial. 

I'm not going to touch the other issues mentioned by @goanpeca and @Nodd to keep things simple :-)

**Comment by ccordoba12** [comment]
@goanpeca, this is ready for a review. I added reasonable contents to all the Intro tour slides.

**Comment by goanpeca** [comment]
Cool, will test it later on :-)

**Comment by ccordoba12** [comment]
@goanpeca, I think this is ready for merge too ;-)

**Comment by goanpeca** [comment]
Go for it!

**Comment by ccordoba12** [comment]
Oki doki! ;-)

**Pull request merged**